src/saving/SaveManagerGUI.cs

```diff
@@ -317,4 +336,30 @@ private void OnSaveListItemConfirmed(SaveListItem item)
     {
         item.LoadThisSave();
     }
+
+    private void OnFilterSelected(int index)
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        switch (index)
+        {
+            case (int)SaveInformation.SaveType.Manual:
+                saveList.Filter(SaveInformation.SaveType.Manual);
+                break;
+            case (int)SaveInformation.SaveType.AutoSave:
+                saveList.Filter(SaveInformation.SaveType.AutoSave);
+                break;
+            case (int)SaveInformation.SaveType.QuickSave:
+                saveList.Filter(SaveInformation.SaveType.QuickSave);
+                break;
+            case 3:
+                saveList.Filter(SaveInformation.SaveType.Manual, true);
+                break;
+        }
```

Comment:
You could do something like this instead.

(You can use `null!` here as that argument is never read if `showAll` is set to true)

```C#
if (index == 3)
{
    saveList.Filter(null!, true);
    return;
}

saveList.Filter((SaveInformation.SaveType)index);
```